DataTables.Blazor.Demo/Pages/Index.razor

```diff
@@ -556,12 +671,20 @@ private record Weather
         }
     };
 
+    private string EventsLogs { get; set; }
+
+    void AddEventLog(string log)
+    {
+        EventsLogs +=  $"- {log}\n";
+        Task.Run(() => JS.InvokeVoidAsync("demo.scrollToBottom", "eventsLogs"));
```

Comment:
Any reason not to make this method async and return the task?

Reply:
I usually use void for event handlers.